### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by caocuong0306** [Joker316701882/Additive-Margin-Softmax#13]
Dear @Joker316701882 ,

Can  you share the Tensorflow pre-trained model, so we can download and play with it. For example, the model trained with VGGFace2.
Thank you very much. Btw, what a great repository!

**Comment by InitAction** [comment]
@Joker316701882 Can you share the Tensorflow pre-trained model?
tanks
